MyProject/cls/Guest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cls
+{
+    public class Guest:Person
+    {
+        public int Visits;
+        public string LoyaltyLevel;
```

Comment:
i would assume that this field should be reworked to property with privte set to make it impossible to change it from outside. Now it seems that LoyalityLevel could be set from anywhere

Reply:
Done, added method to Get the level